error_report/sentry.py

```diff
@@ -96,22 +148,31 @@ def create_sentry_report(report):
         user=dict(id=machine_id),
         fingerprint=[module],
         contexts=get_device_info(report["Environment"][0]),
-        tags=dict()
+        tags=dict(),
+        modules=packages,
     )
-    if module.startswith("orangecontrib"):
-        m = module.split(".")
-        data["tags"]["addon"] = m[1]
-    return data
+    return inspect_for_addons(report["Stack Trace"], data)
 
 
-def send_to_sentry(report, dsn):
-    sentry_report = create_sentry_report(report)
-    if not sentry_report:
+def send_to_sentry(report):
+    sentry_reports = create_sentry_report(report)
+    if not sentry_reports:
         return
 
-    try:
-        client = Client(dsn)
-        client.send(**sentry_report)
-    except Exception as ex:
-        # There is nothing we can do if sentry is not available
-        logger.exception(ex)
+    def send(sentry_dsn, sentry_report):
+        try:
+            client = Client(sentry_dsn)
+            client.send(**sentry_report)
+        except Exception as ex:
+            # There is nothing we can do if sentry is not available
+            logger.exception(ex)
+
+    # send reports
```

Comment:
I would refactor this into something like:

convert to sentry format,
call a function that generates pairs (DSN, report), and then send those reports